lib/pure/strutils.nim

```diff
@@ -154,6 +154,59 @@ proc isDigit*(c: char): bool {.noSideEffect,
     doAssert isDigit('8') == true
   return c in Digits
 
+proc isNumeric*(s: string): bool =
+  ## Check whether the string is numeric.
+  ## When the string is an integer, float or exponential, it returns true,
+  ## otherwise it returns false.
+  ## Note: The reason why `parseFloat()` is not used to achieve this is its
```

Comment:
If this note should be part of public documentation, add one blank comment line before it, and bold it:

```suggestion
  ##
  ## **Note:** The reason why `parseFloat()` is not used to achieve this is its
```